### PR TITLE
unify strings to "payment". Add retry on balance update

### LINKED ISSUE #2733: Balance sometimes does not refresh after signing a TX
and sometimes shows '0'

### PR DESCRIPTION
**Pull request by matiu**
-  Unifies texts: sign -> approve, spend proposal -> payment proposal,   transaction -> payment.
 - Fixes #2733

### CODE REVIEW

File: src/js/services/notificationsService.js [hunk 1]
```diff
@@ -57,23 +57,23 @@ angular.module('copayApp.services')
...
             name, {color: color} );
           break;
         case 'TxProposalRejectedBy':
-          notification.error(gettext('Transaction Rejected'),
+          notification.error(gettext('Payment Proposal Rejected by Copayer'),
             name, {color: color} );
           break;
         case 'TxProposalFinallyRejected':
-          notification.error(gettext('A transaction was finally rejected'),
+          notification.error(gettext('Payment Proposal Finally Rejected'),
```
Comment (martindale) [inline src/js/services/notificationsService.js]: What is this particular message used for?  "Finally" has a sarcastic tone to it, maybe it should be something like: "Payment Proposal was Rejected and Closed"

Comment (matiu) [inline src/js/services/notificationsService.js]: Thanks!

**Comment by matiu** [comment]
updated with @martindale comments.

**Comment by cmgustavo** [comment]
Ack. Manual test: ok

**Pull request merged**